src/views/examples/typography/index.njk

```diff
@@ -111,7 +111,21 @@
       <p class="govuk-body-l govuk-!-w-bold">govuk-body-l + govuk-!-w-bold</p>
 
     </section>
-
+    <section class="govuk-!-mt-r8">
+      <h2 class="govuk-heading-l govuk-!-pb-r2" style="border-bottom: 4px solid;">Lists</h2>
```

Comment:
Should we have an example of a list with neither bullets nor numbers?

Reply:
I think only if we actually do that on gov.uk anywhere - I'll have a look around

Reply:
Lists of links in the related widget tend not to use bullets.

Reply:
thanks - adding now